### PR TITLE
New Data Source:  `azurerm_automation_account`

### PR DESCRIPTION
**Pull request by whytoe**
This Data provider exposes the Keys property in Azure Portal for Automation Account, it uses the goSDK function ".Automation.AgentRegistrationInfoClient" in the ARM client.

I wouldnt be opposed to renaming the data source to be more descriptive but I thought it might be a good idea to expand on this data source for all the automation account properties instead of maintaining multiple data providers for the same resource.

### CODE REVIEW

File: website/azurerm.erb [hunk 1]
```diff
@@ -286,10 +290,6 @@
                     <a href="/docs/providers/azurerm/d/policy_definition.html">azurerm_policy_definition</a>
                 </li>
 
-                <li>
-                    <a href="/docs/d/postgresql_server.html">azurerm_postgresql_server</a>
```
[the remark this answers is not in the document]
Comment (whytoe) [reply website/azurerm.erb]: good catch, not sure how that happened, a rogue vim delete maybe

File: website/azurerm.erb [hunk 2]
```diff
@@ -614,10 +614,6 @@
                   <a href="/docs/providers/azurerm/r/automation_dsc_nodeconfiguration.html">azurerm_automation_dsc_nodeconfiguration</a>
                 </li>
 
-                <li>
-                  <a href="/docs/providers/azurerm/r/automation_job_schedule.html">azurerm_automation_job_schedule</a>
```
[the remark this answers is not in the document]
Comment (whytoe) [reply website/azurerm.erb]: Checked out original azurerm.erb and added data provider for my resource

**Comment by whytoe** [comment]
hey @mbfrahry should this be included in the milestones so it makes it into the changelog? i believe it resolved #2162 as well

**Comment by mbfrahry** [comment]
Good call @whytoe. It looks like this does close that issue as well. Thanks!

**Pull request merged**